server/config/config.js

```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+
+module.exports = {
```

Comment:
Why not use es6 exports?

Reply:
Sequelize does not support the es6 imports and exports.